build.gradle.kts

```diff
@@ -20,7 +20,7 @@ plugins {
   application
 }
 // eCommerce commons library version
-val ecommerceCommonsVersion = "1.10.1"
+val ecommerceCommonsVersion = "1.10.2"
```

Comment:
update gradle.lockfile too

Reply:
Fixed in [fd1376d](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/90/commits/fd1376d470414748aea5b3b40bffd880337fc297)